.github/workflows/maven.yml

```diff
@@ -9,8 +9,14 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - name: Fetch repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
     - run: git fetch origin master
+    - name: Get latest release commit ID
+      id: latest_release
+      run: echo "::set-output name=commit_id::$(git log --pretty=format:"%h%x09%s" -n 1000 | grep "prepare release" | head -1 | cut -f1)"
```

Comment:
Does it mean that the comparison is done only relative to the last commit? Does it take into consideration all squashed commits of this PR relative to master?

Reply:
It will work the same as before.
* Compare the master branch to the last commit of this PR (so it will include all changes of this PR).
* Compare the last released commit to this PR.